### DISCUSSION TITLE
Where is docker-compose.amd.yaml?

### DISCUSSION THREAD

**Discussion by krisives** [hlky/stable-diffusion-webui#1835] [Q&A]
The GitHub pages documentation references `docker-compose.amd.yaml`:

> There is tentative support for AMD GPUs through docker which can be enabled via docker-compose.amd.yml, although this is still in the early stages. Right now, this only works on native linux (not WSL2) due to issues with AMDs support of GPU passthrough. You also must have ROCm drivers installed on the host.

https://github.com/Sygil-Dev/sygil-webui/blob/d88fa9e8c4d9cefbbfb0b445ad79d4ddb85c8e36/docs/1.Installation/4.docker-guide.md?plain=1#L70

However, I cannot find that specific file in the repository and there is no well known specific file with this name available.

Does anyone know?

**Comment by yiqiangjun** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
